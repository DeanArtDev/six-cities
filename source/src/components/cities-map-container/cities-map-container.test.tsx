import {WrappedComponent as CitiesMapContainer} from './cities-map-container';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

describe(`<CitiesMapContainer/>: `, () => {

  it(`should render correctly`, () => {
    const tree = renderer
      .create(
          <CitiesMapContainer/>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
