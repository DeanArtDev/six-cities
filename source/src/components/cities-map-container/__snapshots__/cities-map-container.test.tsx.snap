// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesMapContainer/>:  should render correctly 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
